Source: maxice8
Upstream: no
Reason: fixes compilation with musl libc, and avoid using an internal implementation

--- input/fifo.c
+++ input/fifo.c
@@ -1,6 +1,7 @@
 #include <unistd.h>
 #include <sys/stat.h>
 #include <sys/fcntl.h>
+#include <sys/types.h>
 #include <time.h>
 #include "fifo.h"
 
@@ -27,7 +28,7 @@ void* input_fifo(void* data)
 	int t = 0;
 	//int size = 1024;
 	int bytes = 0;
-	__int16_t buf[BUFSIZE / 2];
+	int16_t buf[BUFSIZE / 2];
 	struct timespec req = { .tv_sec = 0, .tv_nsec = 10000000 };
 
 
